### DISCUSSION TITLE
Support for css @import and @property

### DISCUSSION THREAD

**Discussion by cs-clarence** [vanilla-extract-css/vanilla-extract#1102] [Ideas]
It would be nice if you could use @import with vanilla extract for external global css files (e.g. google fonts, material symbols).

As for @property, right now, only firefox doesn't have support for this.

[https://developer.mozilla.org/en-US/docs/Web/CSS/@property](https://developer.mozilla.org/en-US/docs/Web/CSS/@property)

**Comment by aspirisen** [comment]
About `@property` there is a PR for it https://github.com/vanilla-extract-css/vanilla-extract/pull/1092
